sdk/core/azure-core/tests/test_utils.py

```diff
@@ -134,3 +135,14 @@ def test_get_running_async_module_sync():
         sys.modules.pop("trio", None)
         with pytest.raises(RuntimeError):
             get_running_async_lock()
+
+
+def test_parse_retry_after():
```

Comment:
should the retry policy tests also be updated with float values?

Reply:
Yes